rholang/src/main/scala/coop/rchain/rholang/interpreter/Env.scala

```diff
@@ -0,0 +1,26 @@
+package coop.rchain.rholang.interpreter
+
+object Env {
```

Comment:
As opposed to a trait with a companion? It's grouped in it's own object so that there's somewhere to put `implicit class DeBruijn`. I did that to extend `HashMap` functionality and keep dot notation.

Reply:
We want to extend `HashMap` with additional functionality. We call a hashmap with these additional functions an environment. Since, you can't directly extend a collection in scala, we introduce an implicit class to group the additional functions. This let's us keep dot syntax. Since an implicit class can't be top level, we put it in the `Env` object.